jobserver/settings.py

```diff
@@ -259,6 +262,16 @@
 X_FRAME_OPTIONS = "SAMEORIGIN"
 
 
+# CSP
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_DEFAULT_SRC = ["'self'"]
```

Comment:
From a bit of random clicking around the site, I think localhost will need to be explicitly allowed somewhere for viewing the L4 outputs

Reply:
Hmm, that might be tricky. Env vars would be easier to deal with. I'm not sure I have a nice solution. My ideas are 
1) hard code the release-hatch values in this file and live with the duplication (how often do we add new backends?)
2) use '*' for the relevant section; is it script-src? Then, tidy it up once we have something that works. I don't like the idea of doing something that's insecure, just in case we don't come back to it. However, it's probably better than not having CSPs at all